homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -182,7 +182,9 @@ def _convert_forecast(self, entry):
     @staticmethod
     def _fmt_dewpoint(dewpoint):
         if dewpoint is not None:
-            return round(dewpoint / 100, 1)
+          dewpoint = dewpoint - 273.15
+          return self.hass.config.units.temperature(dewpoint, TEMP_CELSIUS)
```

Comment:
Should we keep the rounding?

Reply:
We need to import `TEMP_CELSIUS` from `homeassistant.const`.

Reply:
Looks like we're missing a couple of spaces in the indentation here too.

Reply:
Removed the rounding and will import the TEMP_CELSIUS. Currently fighting a bit with "self" being not defined, but another task to learn more ;)

Reply:
I think we can keep the rounding. Is there a reason to remove it?

Reply:
ok, lets keep it.